README.md

```diff
@@ -1 +1,76 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> This is a Mobile first Portfolio project for first two sections of Microvere Figma template 3
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo
+
+[Hello Microverse](https://usamamaliq.github.io/Portfolio/)
+
+
+## Getting Started
+
+**A day 3 project of mobile first approach for a figma template
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+HTML & CSS
+
+### Setup
+
+You should have a code editor 
+
+### Install
+
+Install Linters and code editor
+
+### Usage
+
+N/A
+
+### Run tests
+
+To run tests you can use the following commands
+For HTML you should run " npx hint . " on your terminal in root directory of project
+For CSS you should run " npx stylelint "**/*.{css,scss}" " on your terminal in root directory of project
+
+### Deployment
+
+N/A
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@usamamaliq](https://github.com/usamamaliq)
+- Twitter: [@usamamaliq](https://twitter.com/usamamaliq)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/usamamaliq)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse 2nd Day Project
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] It is great that you have added a link to your LICENSE file but where is the LICENSE file in your project. :smile: Please add a license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository#:~:text=On%20GitHub.com%2C%20navigate%20to,click%20Choose%20a%20license%20template.) :+1: